inspirehep/records/api/authors.py

```diff
@@ -5,6 +5,8 @@
 # inspirehep is free software; you can redistribute it and/or modify it under
 # the terms of the MIT License; see LICENSE file for more details.
 
+from inspirehep.records.marshmallow.authors import AuthorsMetadataRawAdminSchemaV1
```

Comment:
why not relative as the other imports?

Reply:
becuase it's not relative on `literature`